src/content/moment/dates/format/notes.md

```diff
@@ -0,0 +1 @@
+[mdn](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/entries)
```

Comment:
This shouldn't be about [`Intl`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl) instead of `Object.entries()`?

Reply:
🤦‍♂️